hooks/pre-commit

```diff
@@ -63,8 +63,12 @@ modified_staged_scripts = to_string(subprocess.check_output(
 added_deleted_staged_scripts = to_string(subprocess.check_output(
     "git diff --cached --name-only --diff-filter=AmD scripts", shell=True))
 
+if added_deleted_staged_scripts != "":
+	print("\nThe legacy scripts of Retriever cannot be added to or removed. To add scripts,\
+	use the retriever-recipes repository.")
+	exit(1)
```

Comment:
Avoid using exit, instead return would be better. exit(1) will cause the tests to fail in later edge cases especially in the Retriever.Julia package and in the rdataretriever package.

Reply:
Would do so.